src/main/java/uk/co/caprica/picam/CameraConfiguration.java

```diff
@@ -288,7 +287,7 @@ public CameraConfiguration captureTimeout(Integer captureTimeout) {
     /**
      * Create a new camera based on the configuration.
      *
-     * @return
+     * @return The constructed camera.
```

Comment:
It's not common convention to capitalise sentences nor use full-stop in Javadoc param or return annotations IMO